repos/system_upgrade/el7toel8/actors/setuptargetrepos/tests/test_setuptargetrepos.py

```diff
@@ -0,0 +1,44 @@
+from leapp.models import CustomTargetRepository, TargetRepositories, RepositoryData, \
+    RepositoryFile, RepositoriesFacts, RepositoryMap, RepositoriesMap
+
+
+def test_minimal_execution(current_actor_context):
+    current_actor_context.run()
+
+
+def test_custom_repos(current_actor_context):
+    custom = CustomTargetRepository(uid='rhel-7-server-rpms',
```

Comment:
I know this is just a test and the content is not that important. But in case somebody wants to get inspiration from tests about the use of some functionality, as I sometimes do, I'd change the RHEL 7 to RHEL 8 repos. Because I was a bit confused as to why to enable RHEL 7 repository during the upgrade to RHEL 8.